phpstan.neon.dist

```diff
@@ -4,13 +4,13 @@ parameters:
 	level: 8
 	treatPhpDocTypesAsCertain: false
 	paths:
-		- load.php
-		- includes/
-		- plugins/
-		- tests/
+		- plugins
+		- tests
+		- performance.php
+		- plugins/performance-lab/load.php
```

Comment:
This line is redundant:

```suggestion
```

Reply:
Interesting. I don't get that error with that line removed.

```
$ npm run phpstan

> phpstan
> composer phpstan

> phpstan analyse --memory-limit=2048M
Note: Using configuration file /home/westonruter/repos/performance/phpstan.neon.dist.
 107/107 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%


                                                                                                                        
 [OK] No errors                                                                                                         
                                                                                                                        
```

Reply:
Interesting. Which version are you on?

Reply:
1.11.1

Reply:
Can you try with `--debug` once since it disabled the cache? Also, it was failing in CI as well - https://github.com/WordPress/performance/actions/runs/9111313232/job/25048167246

Reply:
Humm. No errors when padding `--debug` either.

Reply:
🤔 

No configs override with `phpstan.neon` in the root?

Reply:
Failing in `wp-env` as well with and without XDebug.